src/app/modules/rup/components/ejecucion/hudsBusqueda.component.ts

```diff
@@ -381,6 +385,11 @@ export class HudsBusquedaComponent implements AfterContentInit {
         if (this.ambitoOrigen) {
             this.prestaciones = this.prestaciones.filter(p => p.data.solicitud.ambitoOrigen === this.ambitoOrigen);
         }
+        if (this.organizacionSeleccionada) {
+
+            this.prestaciones = this.prestaciones.filter(p => p.data.ejecucion.organizacion.id === this.organizacionSeleccionada.id);
```

Comment:
```suggestion
            this.prestaciones = this.prestaciones.filter(p => p.data.ejecucion?.organizacion.id === this.organizacionSeleccionada.id);
```
Ya que tal vez no tengan info en ejecucion.organizacion (ejemplo con inicio en top) y te va a pinchar sino.

Reply:
@sychus corregido